### PR TITLE
Creating a Data Model

### PR DESCRIPTION
**Pull request by fickryiman**
- Create a new rails app.
- Initialize the project with Git.
- The project has Postgres database set up.
- Build the project schema.
- Create and run the necessary migration files.
- Table and column names matches the ERD diagram.
- Photo for users table should be a link to an image Foreign keys should be included.
- posts table the column holding the foreign key to the users table should be named author_id
- All columns that are foreign keys have a corresponding index.
- Reference the table or column names or data relationships.

### CODE REVIEW

**Review by Qoosim — COMMENTED** [review]
#  *Status: Invalid Request* :red_circle: 

Hi @fickryiman, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

You have submitted a project, that has linter error. Kindly fix the error and ask for a new review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+    - Table and column names should match the ERD diagram.
+      - note: photo for users table should be a link to an image
+    - Foreign keys should be included.
+      - Pay attention that in `posts` table the column holding the foreign key to the `users` table should be named `author_id`
+    - All columns that are foreign keys should have a corresponding index.
+  - Be sure to reference the from the ERD.
+    <p align="center">
+      <img src="./public/blog_app_erd_v1_1.png" alt="Data model" width="500px" />
+    </p>
+
```
Comment (LionRouge1) [inline README.md:78]: - [ ] There are several broken links in your README file which make it look unprofessional. Kindly make sure that you push the image that you're referring to your repo or simply remove the links from your README file.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+  - Implements Rails naming conventions.
+  - Use params from browser requests in a safe way.
+  - create all controllers and views, won't be processing any data yet, will add simple placeholders for the views with plain HTML.
+  - NOT use rails generate scaffold command. The point of this project is to show how to set up controllers by ourself.
+  - the app handles the URLs used in the wireframes:
+    <p align="center">
+      <img src="./public/blog_root_users.png" alt="All users page wireframe"  width="200px"  />
+      <img src="./public/blog_user_page.png" alt="Single user page wireframe"  width="200px"  />
+      <img src="./public/blog_user_all_posts.png" alt="All posts by a given user page wireframe"  width="200px"  />
+      <img src="./public/blog_single_post.png" alt="Single post page wireframe"  width="200px"  />
```
Comment (LionRouge1) [inline README.md:113]: - [x] Same here as pointed out earlier :point_up_2: These images are not showing. Kindly make sure you have the image in your repo or simply remove them from your README.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+
+This work is licensed under a
+[Creative Commons Attribution 4.0 International License][cc-by].
+
+[![CC BY 4.0][cc-by-image]][cc-by]
+
+[cc-by]: http://creativecommons.org/licenses/by/4.0/
+[cc-by-image]: https://i.creativecommons.org/l/by/4.0/88x31.png
+[cc-by-shield]: https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg -->
+<!-- Creative Commons License - End -->
```
Comment (LionRouge1) [inline README.md]: - [x] Kindly remove all commented out line from your README file. They make it look unprofessional and unclean

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @fickryiman, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Use PostgreSQL adapter for the database :heavy_check_mark: 
- Create migration for different tables :heavy_check_mark: 
- README file looks professional and clean :x:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (fickryiman) [reply README.md]: removed comment

**Review by DammyShittu — COMMENTED** [review]
Hi ,

## APPROVED 🟢

## Highlights
✔️ Creation of tables
✔️ Good documentation

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**